api/pom.xml

```diff
@@ -26,6 +26,16 @@
             <groupId>io.vertx</groupId>
             <artifactId>vertx-web</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-csv</artifactId>
+        </dependency>
```

Comment:
Why is this addition required?

Reply:
I'm using it in csv serializer as an alternative to using reflection